src/py/aspen/cli/db.py

```diff
@@ -59,12 +59,64 @@ def set_passwords_from_secret(ctx):
         db_interface.engine.execute(f"""ALTER USER {username} PASSWORD '{password}'""")
 
 
+@db.command("setup")
```

Comment:
I don't know if "setup" is a great name, but "create" was already taken. This creates a new (empty) db and then loads a SQL file from S3 into it.

Reply:
could we do `create-remote` maybe?

Reply:
nevermind